app/main.py

```diff
@@ -1,6 +1,40 @@
+import requests
+
+import os
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    params = {
+        "key": os.getenv("WEATHER_API_KEY"),
+    }
+
+    get_weather_in_paris_url = (
+        "http://api.weatherapi.com/v1//"
+        "current.json?q=Paris"
+    )
+
+    response = requests.get(get_weather_in_paris_url, params=params)
+    weather = response.json()
+
+    if weather:
+        location = weather.get("location")
+        current = weather.get("current", {})
+        print(
+            f"Country: "
+            f"{location["country"]}"
+            f" - City: {location["name"]}"
+            f" - Region: {location["region"]} | "
+            f" {location["localtime"]}"
+        )
+        print(f"Temperature: "
+              f" {current["temp_c"]}"
+              f" celsius")
```

Comment:
Similar to the previous comment, the string formatting inside this `print` function is incorrect. Use single quotes inside the f-string or escape the double quotes. For example, `f"Temperature: {current['temp_c']} celsius"`.